puppet-modulebuilder.gemspec

```
@@ -25,9 +25,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   # minitar and pathspec is required for building Puppet modules
   spec.add_runtime_dependency 'minitar', '~> 0.9'
-  spec.add_runtime_dependency 'pathspec', '>= 0.2.1', '< 3.0.0'
+  spec.add_runtime_dependency 'pathspec', '~> 2.1'
 
   spec.add_development_dependency 'voxpupuli-rubocop', '~> 2.8.0'
-
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.7.0')
+  spec.required_ruby_version = Gem::Requirement.new('>= 3.1.0')
```

You can just write this:
```suggestion
  spec.required_ruby_version = '>= 3.1'
```